content/src/service/pointers/pointers.ts

```diff
@@ -25,17 +22,12 @@ export async function getPointerChanges(
   )
 
   deploymentsWithExtra = deploymentsWithExtra.filter((result) => !components.denylist.isDenylisted(result.entityId))
+  const moreData: boolean = deploymentsWithExtra.length > curatedLimit
+  const deployments: HistoricalDeployment[] = deploymentsWithExtra.slice(0, curatedLimit)
 
-  const moreData = deploymentsWithExtra.length > curatedLimit
-
-  const deployments = deploymentsWithExtra.slice(0, curatedLimit)
-  const deploymentIds = deployments.map(({ deploymentId }) => deploymentId)
-  const deltasForDeployments = await getPointerChangesForDeployments(components, deploymentIds)
   const pointerChanges: DeploymentPointerChanges[] = deployments.map(
-    ({ deploymentId, entityId, entityType, localTimestamp, authChain }) => {
-      const delta = deltasForDeployments.get(deploymentId) ?? new Map()
-      const changes = transformPointerChanges(entityId, delta)
-      return { entityType, entityId, localTimestamp, changes, authChain }
+    ({ entityId, entityType, localTimestamp, pointers, authChain }) => {
+      return { entityId, entityType, localTimestamp, pointers, authChain }
```

Comment:
Some here, seems that it is mapping to the same thing

Reply:
done